### DISCUSSION TITLE
Math subscript problems with symbols

### DISCUSSION THREAD

**Discussion by egri-nagy** [typst/typst#3568] [General]
### Description

Subscript does not work as expected for the subset symbol. When using `#sub` the subscript is too far from the main symbol. Same for calligraphic letters.

### Reproduction URL

https://typst.app/project/rifeLip1OD135R7-fwbDwt

### Operating system

macOS

### Typst version

- [X] I am using the latest version of Typst

**Comment by Enivex** [comment]
The sub function is intended for use in text, you should not use it in math mode. You're looking for https://typst.app/docs/reference/math/attach#functions-attach (though it's rarely necessary to call it directly)

However, it seems like your _actual_ issue is that you want to change a symbol from displaying with limits to scripts. You can do this with https://typst.app/docs/reference/math/attach#functions-scripts

**Reply by egri-nagy** [reply]
Thank you very much for your help! Calling `attach` did solve my first issue of indexing the subset symbol.

However, the with the calligraphic symbol, the index, does not appear in the correct distance from the main symbol.
Please see second half of the problem, same file.

https://typst.app/project/rifeLip1OD135R7-fwbDwt

**Reply by Enivex** [reply]
That's a combination of the font being suboptimal, and mathcal T being the absolute worst case scenario for scripts.

**Reply by Enivex** [reply]
P.S. you can use _ and ^ instead of calling attach directly. (Combined with usings scripts() or limits() to force specific behavior for symbols)
